script/setup/addNewProject.ts

```diff
@@ -263,6 +269,50 @@ export const getProjectTokenListInfo = async (
         choices: tokenChoices,
       },
     ]);
+    const tokens = response.tokens as Tokens[];
+    const isEthAndWethPresent =
+      tokens.includes(Tokens.ETH) && tokens.includes(Tokens.WETH);
+    const isUsdcAndUsdcePresent =
+      tokens.includes(Tokens.USDC) && tokens.includes(Tokens.USDCE);
+
+    let mergeEthAndWeth = false;
+    let mergeUsdcAndUsdce = false;
+    let mergeInboundWithTokens: {
+      [key in Tokens]?: Tokens[];
+    } = {};
```

Comment:
mergeInboundWithTokens is also assigned outer objected only once so it can be declared as const